src/SpectatorView.Unity/Assets/SpectatorView/Scripts/ScreenRecording/AndroidRecordingService.cs

```diff
@@ -9,8 +9,7 @@ namespace Microsoft.MixedReality.SpectatorView
     /// <summary>
     /// Class implementing <see cref="IRecordingService"/> for the Android platform
     /// </summary>
-    public class AndroidRecordingService : MonoBehaviour,
-    IRecordingService
+    public class AndroidRecordingService : IRecordingService
```

Comment:
>AndroidRecordingService [](start = 17, length = 23)

If you think that this is something that has a chance of being extended, adding a release note for this would be helpful.

Likelihood may be low so it's probably fine to also not have anything mentioned